contracts/integrations/test/framework/balances/types.ts

```diff
@@ -6,7 +6,7 @@ import { BigNumber } from '@0x/utils';
 // alias for clarity
 type address = string;
 
-interface TokenData<TERC20, TERC721, TERC1155> {
+export interface TokenData<TERC20, TERC721, TERC1155> {
```

Comment:
Can we actually skip exporting everything in this file? I grepped and it looks like they're not used anywhere. We're avoiding importing `contracts/integrations` into anything because it's supposed to be used to resolve dependency cycles.

Reply:
Yep, will fix.